src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/css/app.css

```diff
@@ -63,6 +63,10 @@ a, .btn-link {
         content: "An error has occurred."
     }
 
+:root {
+    --blazor-load-percentage: 0;
+}
+
```

Comment:
```suggestion
```

Good trick, but this is not needed - can use the native fallback as per other suggestion.